res/xml/bone.xml

```diff
@@ -0,0 +1,45 @@
+<?xml	version="1.0"	encoding="utf-8"?>
+<keyboard>
+	<row>
+		<key   width="1"   key0="j"							key2="°"						key4="esc"/>
+		<key   width="1"   key0="d"							key2="§"						key4="_"/>
+		<key   width="1"   key0="u"							key2="ℓ"						key4="["/>
+		<key   width="1"   key0="a"							key2="»"						key4="]"/>
+		<key   width="1"   key0="x"							key2="«"						key4="^"/>
+		<key   width="1"   key0="p"			key1="$"						key3="!"		key4="7"/>
+		<key   width="1"   key0="h"			key1="€"		key2="⟨"		key3="&lt;"		key4="8"/>
+		<key   width="1"   key0="l"			key1="£"		key2="⟩"		key3="&rt;"		key4="9"/>
+		<key   width="1"   key0="m"			key1="¥"						key3="="		key4="f11_placeholder"/>
+		<key   width="1"   key0="w"			key1="¢"						key3="&amp;"	key4="f12_placeholder"/>
+	</row>
+	<row>
+		<key   width="1"   key0="c"							key2="tab"						key4="\"/>
+		<key   width="1"   key0="t"			key1="accent_circonflexe"	key2="accent_caron"	key4="/"/>
+		<key   width="1"   key0="i"			key1="accent_aigu"			key2="accent_grave"	key4="{"/>
+		<key   width="1"   key0="e"			key1="accent_cedille"	key2="accent_ogonek"	key4="}"/>
+		<key   width="1"   key0="o"			key1="accent_ring"		key2="accent_dot_above"	key4="*"/>
+		<key   width="1"   key0="b"					key2="accent_macron"	key3="?"		key4="4"/>
+		<key   width="1"   key0="n"					key2="accent_tilde"		key3="("		key4="5"/>
+		<key   width="1"   key0="r"					key2="accent_trema"		key3=")"		key4="6"/>
+		<key   width="1"   key0="s"					key2="accent_slash"		key3="-"				/>
+		<key   width="1"   key0="g"			key1="@"						key3=":"				/>
```

Comment:
Some characters need to be escaped due to Android's XML having all kinds of new syntaxes (references to resources, colors). Same for `?` and `#`.
```suggestion
		<key   width="1"   key0="g"			key1="\@"						key3=":"				/>
```